conformance/reports/v1.1.0/nginxinc-nginx-gateway-fabric/experimental-1.4.0-default-report.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: gateway.networking.k8s.io/v1alpha1
+date: "2024-08-20T18:43:28Z"
+gatewayAPIChannel: experimental
+gatewayAPIVersion: v1.1.0
+implementation:
+  contact:
+  - https://github.com/nginxinc/nginx-gateway-fabric/discussions/new/choose
+  organization: nginxinc
+  project: nginx-gateway-fabric
+  url: https://github.com/nginxinc/nginx-gateway-fabric
+  version: v1.4.0
```

Comment:
Tiny nit: Would be great to have a corresponding update to readme.md and implementations.md

Reply:
What change are you looking for in implementations.md? I don't see our version listed there.

Reply:
Usually link to the latest conformance report from both.

Reply:
Ours just links to the top level nginx conformance directory, not a specific report.